AUTHORS

```diff
@@ -2,22 +2,23 @@ OpenRA wouldn't be where it is today without the
 hard work of many contributors.
 
 The OpenRA developers are:
-    * Chris Forbes (chrisf)
+    * Gustas Kažukauskas (Punsho)
```

Comment:
```suggestion
    * Gustas Kažukauskas (PunkPun)
```

Please update it to my new github name 🙌

Reply:
Ah right.